zjb/templates/results.html.j2

```diff
@@ -36,4 +39,83 @@
                 </tfoot>
             </table>
             {%- endfor -%}
+
+            <script>
+                function openTextbox(project, branch, pipeline, job) {
```

Comment:
An important feedback for me is whether in general you are ok with creating a dialog box when clicked on ... button, having 3 objects on that dialog (text, save button, close button), fetching from a database using '/fetch-from-database', methods=['GET'] and populating textbox.value. Then, if save button was clocked saving into the database using '/save-to-database', methods=['POST']

Reply:
I general, I like the idea, but this implementation seems a little over-complicated for my taste... Let me browse for a while, check how it works and maybe I will be able to simplify this a little :man_technologist: 

Nevertheless, thank you for this draft change! :smiley_cat: